### PR TITLE
Error handling for issue #279

### LINKED ISSUE #279: Ask search engine generating error for some special case
**I'm submitting a ...**
- [x] bug report
- [ ] feature request

**Current behavior:**

type 'a b c d e f' and then you will notice that it will give 500 error

Error is generating because as you can see there are 4 videos in one div and there are no descriptions.
This can be possibly solve by removing description tag from results and show only url and title. 

screenshot from 2017-10-26 23-33-54

### PR DESCRIPTION
**Pull request by Dragneel7**
Fixes #279

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

-
-

### CODE REVIEW

File: app/scrapers/ask.py [hunk 1]
```diff
@@ -21,11 +21,14 @@ def parseResponse(self, soup):
...
         if soup.find('div', {'class': 'PartialSearchResults-noresults'}):
             return None
+          
         for div in soup.findAll('div', {'class': 'PartialSearchResults-item'}):
             title = div.div.a.text
             url = div.div.a['href']
-            p = div.find('p', {'class': 'PartialSearchResults-item-abstract'})
-            desc = p.text.replace('\n', '')
-            urls.append({'title': title, 'link': url, 'desc': desc})
-        print('Ask parsed: ' + str(urls))
```
Comment (rupav) [inline app/scrapers/ask.py]: This statement got removed, should remain in the code.

**Comment by Dragneel7** [comment]
@shashank-sharma Build is failing in python 2.7 because of the unused variable 'e' "https://travis-ci.org/fossasia/query-server/jobs/297287440". How should I handle this?

**Comment by rupav** [comment]
screenshot 2017-11-06 05 51 22

@Dragneel7 try using 
' except Exception: ' instead of bare `except` .

**Comment by Dragneel7** [comment]
Thanks, I will keep that in mind.

**Comment by rupav** [comment]
@Dragneel7 ask atleast 2 maintainers to get it approved.

**Comment by Dragneel7** [comment]
@shashank-sharma  @bhaveshAn Please review the PR.

**Comment by AnshulMalik** [comment]
Please rebase with master and we are good to go @Dragneel7

**Comment by Dragneel7** [comment]
@AnshulMalik I have rebased with master

**Pull request merged**